Runs: Adds doc on customizing and resetting run colors (#1428)
wandb/docs: content/en/guides/models/track/runs/run-colors.md

---
description: 
menu:
  default:
    identifier: run-colors
    parent: Customize run colors
title: Customize run colors
---

> W&B automatically assigns a color to each run that you create in your project. You can change the default color of a run to help you visually distinguish it from other runs in the table and graphs. Resetting your project workspace will restore the default colors for all runs in the table.

Reviewer: ```suggestion
W&B automatically assigns a color to each run in your project. You can change the default color of a run to help you visually distinguish it from other runs in the table and graphs. Reset your project workspace to restore the default colors for all runs in the table.
```

> Run colors are locally scoped. On the project page, custom colors only apply to your own workspace. In reports, custom colors for runs only apply at the section level. You can visualize the same run in different sections, and it can have a different custom color in each section.

Reviewer: ```suggestion
Run colors are locally scoped. On the project page, custom colors apply only to your own workspace. In reports, custom colors for runs apply only at the section level. You can visualize the same run in different sections, which can use different custom colors per section.
```

## Edit default run colors

1. Click the **Runs** tab from the project sidebar.
2. Click the dot color next to the run name in the **Name** column.

> 3. Select a color from the color palette, the color picker, or enter a hex code.

Reviewer: ```suggestion
3. Select a color from the color palette or the color picker, or enter a hex code.
```

{{< img src="/images/runs/run-color-palette.png" alt="Edit default run color in project workspace">}}

## Randomize run colors

To randomize the colors of all runs in the table:

1. Click the **Runs** tab from the project sidebar.
2. Hover over the **Name** column header, click the three horizontal dots (**...**), and select **Randomize run colors** from the dropdown menu.

{{% alert %}}

> The option to randomize run colors is only available if you have made some kind of modification to the run's table (sorting, filtering, searching, or grouping).

Reviewer: ```suggestion
The option to randomize run colors is available only after modify the run's table in some way, such as by sorting, filtering, searching, or grouping.
```
